### ISSUE TITLE
IE11 problems

### ISSUE THREAD

**Issue by Sawadski** [floragunncom/search-guard-kibana-plugin#78]
Hi,
regarding #55, I tested searchguard on the `Internet Explorer 11` in `Windows 7` and found 2 bugs.

1. **Highlighting Bug**
     With this pull request #67 this would have been fixed, but on the release there is an additional change in the [login.html](https://github.com/floragunncom/search-guard-kibana-plugin/blob/6.x/public/apps/login/login.html#L17) file.
     ```
     ... || ! ui.credentials.username" ...
     ```
     Without this condition the highlighting works also on the `IE11`. 
     Thus, is this condition needed?

2. **Custom stylings**
     In my case, I would like to change the color of the login button. I changed the color in the `kibana.yml`:
     ```
     ...
     searchguard.basicauth.login.buttonstyle: "background-color: red;"
     ...
     ``` 
     It works with the newer browser like `Chrome` or `Firefox` but not with `IE11`.
     I think the problem is here [login.html](https://github.com/floragunncom/search-guard-kibana-plugin/blob/6.x/public/apps/login/login.html#L24).
     ```
     ... style="{{ui.buttonstyle}}" ... 
     ```
     The `IE11` does not get the custom settings.

**Comment by floragunncom** [comment]
Thanks, will look into this.

**Comment by mgustafsson1** [comment]
Hi there, 

Just a quick update on this one before I submit a PR.
I've tested in `IE11` on `Windows 7`, and I can reproduce both issues.

1. **Highlighting bug**
I added that extra condition to make sure that the form can't be submitted with empty values.
I'll come up with a smarter solution that works in `IE11` too.

2. **Custom stylings**
You are right, `IE11` seems to strip values from the style attribute that it doesn't recognise as valid css. In this case it seems to be Angular's `{{}}` that's causing the problem.
I'll add support for `IE11`:
    ```
     ... ng-attr-style="{{ui.buttonstyle}}" ... 
     ```
I'll submit a PR to fix these issues.
